Project 8 - Mapty/css/style.css

```diff
@@ -0,0 +1,319 @@
+/* CSS for Mapty */
+:root {
+  --color-brand--1: #45ecff;
+  --color-brand--2: #00c46a;
+
+  --color-dark--1: #2d3439;
+  --color-dark--2: #42484d;
+  --color-light--1: #aaa;
+  --color-light--2: #ececec;
+  --color-light--3: rgb(214, 222, 224);
```

Comment:
@sknepali You can convert all rgb & rgba into hex for consistency.

Reply:
okay sir